Meeting notes link update

klumb opened the pull request.

jgbarah commented on meeting_notes.md:
```diff
@@ -7,22 +7,81 @@ The CHAOSS Growth Maturity and Decline working group meets every other Wednesday
 
------------
+WG Repository: https://github.com/chaoss/wg-evolution
+Notes: https://docs.google.com/document/d/1fgMT5onwvNQE6b4gPWE7oSPHRvb9q1z6XEbD51EtCFg/edit#
```
Same as above.

jgbarah commented on README.md:
```diff
@@ -12,7 +12,7 @@ The Evolution Working Group meets every Monday from 7:30am - 8:30am Pacific Time
 The videoconference URL is https://unomaha.zoom.us/j/720431288.
 
-[You can also read our meeting notes.](https://docs.google.com/document/d/1Owsn8B7ciPPUHHypYiFAe0Bps58QGGYSVrqHNX2l8p8)
+[You can also read our meeting notes.](https://docs.google.com/document/d/1fgMT5onwvNQE6b4gPWE7oSPHRvb9q1z6XEbD51EtCFg/edit#heading=h.eqofqt9ml5ad)
```
Instead of this, i suggest that we use the link to the file in this repository, since that's the final version of the notes:

```
(meeting_notes.md)
```

Then, I would also add the link to the Google Docs document, labeled as (see also temporary version, used to create notes during the meetings).

klumb replied:
I think we can leave the google doc off the readme. I will include it in the meeting_notes.md file though.

jgbarah commented:
@kevin, I see a new push, but my comments were not addressed. If that's your preference, please comment, and I will accept the change, and later propose the change as I suggest in a separate pr.

klumb commented:
Sorry, I hadn’t seen your comments until after the push. I am looking at them now.

Cheers, 


Kevin Lumbard (he/him/his)
University of Nebraska at Omaha
PKI 367

> On May 9, 2019, at 10:36 AM, Jesus M. Gonzalez-Barahona <notifications@github.com> wrote:
> 
> @kevin <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_kevin&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=Issqq6yO-ovFOx6PfTtqTsYMu1wiJPFnXztcE7Cv-Bs&m=CJJIq-8cDBwNq8GgmnfFvF817XADyNTV2iLgXbb0oJ4&s=90jwOJgDFttoCv16URz3sd9zMehZAnnl1soEX83o7ss&e=>, I see a new push, but my comments were not addressed. If that's your preference, please comment, and I will accept the change, and later propose the change as I suggest in a separate pr.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chaoss_wg-2Devolution_pull_146-23issuecomment-2D490955258&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=Issqq6yO-ovFOx6PfTtqTsYMu1wiJPFnXztcE7Cv-Bs&m=CJJIq-8cDBwNq8GgmnfFvF817XADyNTV2iLgXbb0oJ4&s=3jv0d5XTSEsZoFX_-PRi9NNXGIf1U2Vt9QSgeM696IQ&e=>, or mute the thread <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ACQNBNPY4CZUMIHIBY3RHK3PURAJTANCNFSM4HLSNT5A&d=DwMCaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=Issqq6yO-ovFOx6PfTtqTsYMu1wiJPFnXztcE7Cv-Bs&m=CJJIq-8cDBwNq8GgmnfFvF817XADyNTV2iLgXbb0oJ4&s=EFbZI1ytMyVrRcM4tTadvs6RfLT_lJJqgGsiId1sNMg&e=>.
>

klumb commented:
I agree, the meeting notes link on readme should point to the meeting_notes.md. I made the change. I also changed the wording on the google doc link and hid the URL. I think this addresses your comments. I didn't include a link to the google doc on the readme as I think it may be redundant. If you want to include it I will submit another patch.

GeorgLink commented:
After switching links from the google doc to the GitHub archive, where will people find the link to the google doc when they join a call?

jgbarah commented:
@GeorgLink I think that's in the file:

> We use [Google Notes](https://docs.google.com/document/d/1fgMT5onwvNQE6b4gPWE7oSPHRvb9q1z6XEbD51EtCFg/edit#) to take the meeting minutes. Notes are captured by the entire group and a volunteer acting as the primary scribe.
